packages/jupyter-ai/src/index.ts

```diff
@@ -66,7 +74,9 @@ const plugin: JupyterFrontEndPlugin<void> = {
       });
     };
 
-    let chatWidget: ReactWidget | null = null;
+    const inputFocusRequested = new Signal<unknown, void>({});
```

Comment:
Can you rename this constant & all props that take this constant to `focusInputSignal`? The new name 1) hints that this is a signal object, not a boolean, and 2) is more aligned with the `focusInputElement()` function defined in `chat-input.tsx`.

Reply:
Done in 6df988d146ceab455d2c65b8d0946b5591807cf4.

> Personally I disagree with (1) as this is a common convention to name signals in past tense when indicating that signal is emitted when an action completed, and in TypeScript there is no need to suffix or prefix things with types. That said, since signals are used sparsely in this extension, this is probably fine from the perspective of making the API intuitive.

Reply:
> in TypeScript there is no need to suffix or prefix things with types

Personally, I feel like this depends on context. If it's a primitive type, then I agree. But for custom types like ones imported from Lumino, I think indicating the type in the name itself benefits readability. This way, I don't have to scroll up to the type declaration to determine "what this variable is".

Let's prefer `focusInputSignal` for now since that change is already made. I do see your perspective however.